evaluation/evaluation_engine.py

```diff
@@ -94,15 +101,88 @@ def execute_model(
         )
 
 
-def sacrebleu_score(reference, hypothesis):
-    return sacrebleu.sentence_bleu([hypothesis], [reference]).score
+def convert_ner_ids_to_tags(ner_tags):
+    # convert list of ner ids [0,1,2,0] to list of ner tags ['0', 'B-PER', 'I-PER', '0']
+    ner_tag_sequence = []
+    ner_tag_dict = {1: 'B-PER', 2: 'I-PER', 3: 'B-ORG', 4: 'I-ORG', 5: 'B-LOC', 6: 'I-LOC', 7: 'B-MISC', 8: 'I-MISC'}
+    for tag in ner_tags:
+        if(tag in ner_tag_dict.keys()):
+            ner_tag_sequence.append(ner_tag_dict[tag])
+        else:
+            ner_tag_sequence.append('0') # O(big O), tag for no ner token
+    return ner_tag_sequence
+
+def calculate_seq_evaluation_score(gold, prediction):
+    return accuracy_score([gold], [prediction])
```

Comment:
you can simply call accuracy_score directly.. no need of a new function for just one line